crypto/src/pubkey_bytes.rs

```diff
@@ -67,18 +67,31 @@ impl<T, const N: usize> Default for PublicKeyBytes<T, N> {
     }
 }
 
+impl<T: VerifyingKey, const N: usize> FromStr for PublicKeyBytes<T, N> {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let s = s.strip_prefix("0x").unwrap_or(s);
+        let value = hex::decode(s)?;
+        Self::from_bytes(&value[..]).map_err(|_| anyhow::anyhow!("byte deserialization failed"))
+    }
```

Comment:
Is there a test already covering this ? If not then I would suggest adding one

Reply:
I will improve test coverage on PublicKeyBytes this PR as well.